OpenTabletDriver.Plugin/Platform/Pointer/IVirtualTablet.cs

```diff
@@ -9,5 +9,6 @@ public interface IVirtualTablet : IAbsolutePointer
         void SetButtonState(uint button, bool active);
         void SetEraser(bool isEraser);
         void SetProximity(bool proximity, uint distance);
+        void Finish();
```

Comment:
I would prefer to continue to use sync to describe this functionality.
```suggestion
        void Sync();
```

Reply:
Ok